README.md

```diff
@@ -332,6 +332,15 @@ const container = await new GenericContainer("alpine")
   .start();
 ```
 
+Creating a container with volumes associated:
+
+```javascript
+const { GenericContainer } = require("testcontainers");
+
+const container = await new GenericContainer("alpine")
+  .withVolumes("volumes/new-volume")
```

Comment:
could the example show that you can provide multiple volumes, similar to the `withExposedPorts` example? else people will ask how to specify multiple volumes